### ISSUE TITLE
Block.defineBlock() not really working

### ISSUE THREAD

**Issue by StudioJR** [zhuowei/MCPELauncher#1481]
My Problem is that when I create a Block in the newes Blocklauncher 1.19 patching Mcpe 1.5 the block seems to be added - no errors - and the block appears in creative inventory with the correct Texture, but when i try to place it - it always places a block looking like a stonecutter
E.g..:
Block.defineBlock(3333,"Wooden Slab",["woodenslab",0])

**Comment by XsJIONG** [comment]
The ID of a block is limited between 1 to 255, and most of them are already used, you should choose a proper ID via reading Minecraft ID Map

**Comment by zedman2000** [comment]
@XsJIONG the newest versions can have block IDs that large as a "stacked" kind of thing.  There should not be an issue putting one in that high.
@StudioJR a good test would be to try overwriting some other block, as well as testing on another version, just to ensure that it is a problem with this version.  You might also try defining EVERY side of the block with a complete string.  It is possible that any or none of these are the cause.

**Comment by StudioJR** [comment]
@zedman2000 I can say that I've testet it in an older version, cause it worked in Minecraft 1.2 the Minecraft Version I created it for

**Comment by StudioJR** [comment]
@XsJIONG I'm new to this Stuff can you explain me what a Minecraft ID Map is

**Comment by StudioJR** [comment]
Ok I've tried it again and added my Indicator Mod wich tells me every available Data for the Block so I've found out that the Block not obly looks like a stonecutter it is a stonecutter

**Comment by StudioJR** [comment]
![screenshot_20180807-080018](https://user-images.githubusercontent.com/41320307/43757108-fcfbfb9c-9a17-11e8-93d2-35c93a277748.jpg)
This Screenshot is taken on my phone - I've git my hands on the Pocket Decoration Mod its a little bit old and as far as I know - does this mod use ids from 1 - 255 and you see whats happend with them

**Comment by StudioJR** [comment]
@XsJIONG or do you mean an ID List

**Comment by XsJIONG** [comment]
Yes, there are some grammar mistakes in my comments since I don't speak English usually
